recipes/keystone/all/conanfile.py

```diff
@@ -0,0 +1,101 @@
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.files import (
+    copy,
+    get,
+    rmdir,
+    export_conandata_patches,
+    apply_conandata_patches,
+)
+from conan.tools.microsoft import is_msvc, is_msvc_static_runtime
+from conan.tools.scm import Version
+import os
+
+required_conan_version = ">=1.53.0"
+
+
+class KeystoneConan(ConanFile):
+    name = "keystone"
```

Comment:
Should I dd ?
```version = "0.9.2"```

Reply:
It's not needed to add the `version` attribute in ConanCenterIndex recipes, it's managed by the version listed in conandata.yml, so you can re-use the very same recipe for future versions. 

> Because conan export-pkg wont work without version specification

True, but you can pass the required version by the argument `--version=x.y.z`